### ISSUE TITLE
Additional query for a tag characteristic

### ISSUE THREAD

**Issue by michaelmgis** [riccardoklinger/OSMquery#10]
Example:
highway=primary AND surface=paved

**Comment by riccardoklinger** [comment]
maybe @rastrau can help with this?

**Comment by rastrau** [comment]
I think this could be valuable feature for power users, but I suggest giving it a lower priority compared to e.g. usability and code structure improvements for now.

**Comment by riccardoklinger** [comment]
I created the OverpassQuery branch for this.
@rastrau @michaelmgis please test and provide feedback. looks good to me, so I am ready for merge. commits from customCRS branch should be merged first!

**Comment by riccardoklinger** [comment]
done with new tool in commit a3a076ba6a930e0307f3aab5b761199c068b6ba5
